engine/src/flutter/web_sdk/BUILD.gn

```diff
@@ -420,7 +419,6 @@ if (!is_fuchsia) {
     deps += [ "//flutter/lib/web_ui/flutter_js" ]
 
     # flutter_ddc_modules
-    sources = get_target_outputs(":flutter_dartdevc_canvaskit_kernel_sdk_amd")
     sources += get_target_outputs(":flutter_dartdevc_canvaskit_kernel_sdk_amd")
```

Comment:
Might need to keep the first one because it is the original assignment without the `+=`.

Reply:
Right, that's how it works :P